src/main/app/case/case.ts

```diff
@@ -507,6 +507,10 @@ export interface Case {
   childFirstName?: ChildrenDetails['firstName'];
   childLastName?: ChildrenDetails['lastName'];
   cd_children?: ChildrenDetails[];
+  otherChildFirstName?: ChildrenDetails['firstName'];
```

Comment:
Do we need these two properties (otherChildFirstName  &  otherChildLastName)in case data for other child names? Can't we reuse the temp property created for children?

Reply:
we can but if I change the type for ChildrenDetails to its own type these values wont match up, it will work but they wont match up as long as you dont mind @vivek-sekhar

Reply:
> we can but if I change the type for ChildrenDetails to its own type these values wont match up, it will work but they wont match up as long as you don't mind @vivek-sekhar

Sorry I don't get you. what own type you are referring to? I still can see opportunity to make use of child first name & last name property to reuse.

Reply:
So for childFirstName the type is ChildrenDetails['firstName'] this will be different from OtherChildrenDetails which is the new type I have created for other children, its only referring to type of string but the  wont match, it works but Im just checking your okay with it